code/game/jobs/job/support.dm

```diff
@@ -467,34 +467,6 @@
 			continue
 		H.add_language(la)
 
-/datum/job/barber
```

Comment:
Remove the barber icon_state from sechud.dmi please

Reply:
Done